index.html

```diff
@@ -98,96 +99,102 @@ <h2>The Finance and Investment club of NIT Trichy.</h2>
 
         <div class="section-title">
           <h2>About Us</h2>
-          <p>Magnam dolores commodi suscipit. Necessitatibus eius consequatur ex aliquid fuga eum quidem. Sit sint consectetur velit. Quisquam quos quisquam cupiditate. Et nemo qui impedit suscipit alias ea. Quia fugiat sit in iste officiis commodi quidem hic quas.</p>
+          <p>What started out as a WhatsApp Group among students interested in finance, ProfNITT is now the finance and investments club of NIT Trichy,
+             with more than 30 members.We aim to bring about financial literacy among students of NIT-Trichy, helping them make well-informed
+             monetary decisions.
+          </p>

```

Comment:
Accentuate certain things like "ProfNITT" with bold so that it's easy to read. And you're missing a space after period in `30 members.We`.

Reply:
cool